src/main.cpp

```diff
@@ -107,10 +107,113 @@ static SDispatchResult dispatch_move_window(std::string arg) {
     return {};
 }
 
+static void set_offset(int new_offset) {
+    for (PHTVIEW view : ht_manager->views) {
+        if (view == nullptr)
+            continue;
+        view->layout->first_ws_offset = new_offset;
+        Debug::log(
+            LOG,
+            "[Hyprtasking] offset was: {}, new: {}",
+            ht_manager->offset,
+            new_offset
+        );
+    }
+}
+
+static SDispatchResult dispatch_setoffset(std::string arg) {
+    if (ht_manager == nullptr)
+        return {.success = false, .error = "ht_manager is null"};
+
+    if (arg[0] == '+' || arg[0] == '-') {
+        ht_manager->offset += std::stoi(arg);
+    } else {
+        ht_manager->offset = std::stoi(arg);
+    }
+    set_offset(ht_manager->offset);
+    return {};
+}
```

Comment:
Setting the offset directly doesn't update the currently focused workspace.

Reply:
Correct, the issue is that if you change the offset, the currently visible workspace does not also change. Let's say you're using a 2x2 grid with 2 layers, if you're on the top-left workspace (0) and change the offset to 4, the current workspace is still 0, which isn't available. I would expect the current workspace to now be 4.

Maybe I should add that as a separate command?

Reply:
> > Maybe I should add that as a separate command?
> 
> You mean a jump to a specific workspace?

I'm trying to make it so I can jump to a specific layer, rather than just going +/- relative to the current layer. 0bd61d3441549e6582e9efddca99402a93de2d84 works, so I can just do `hyprtasking:setoffset, 0` to go to the first layer, then `hyprtasking:setoffset, [grid size]` to go to the second layer, etc. Thanks for fixing that!

Reply:
Can confirm, this works perfectly now! Thanks again!